storage_service/common/utils.py

```diff
@@ -690,7 +698,7 @@ def find_tagmanifest(file_path):
 
     for tagmanifest in tagmanifest_files:
         if tagmanifest in bag_files:
-            return os.path.join(file_path, tagmanifest)
+            return file_path / tagmanifest
```

Comment:
I think this also needs to be returned as a string.

Reply:
Could you please take a look at this? The `find_tagmanifest` function is exclusively utilized by the `generate_checksum` function within [`common.utils`](https://github.com/klavman/archivematica-storage-service/blob/478630dd9ebca1c1d8527f6ece229ec53aa50925/storage_service/common/utils.py#L675). Although `generate_checksum` is used in `package.py` or within the `locations` module, its sole purpose is to calculate and return the checksum. 

If I'm missing something, please let me know, and I will make the corresponding change and adjust the involved tests accordingly.

Reply:
Ah I see. Thanks for tracking the call chain. In that case I think returning a `pathlib.Path` is safe :+1: (we're **inside** the "sandwich" :drum: :blush:)